packages/frontend/types/ember-statecharts.d.ts

```diff
@@ -0,0 +1,10 @@
+import { MachineConfig, Interpreter, EventType, EventObject, StateMachine, InterpreterOptions } from "xstate";
+
+export function useMachine<
+  Context = {},
+  Schema = {},
+  Events extends EventObject = TODO<any>
+>(
+  machineConfig: MachineConfig<Context, Schema, Events>,
+  options: Partial<InterpreterOptions>
+): StateMachine<Context, Schema, Events>;
```

Comment:
accessing the usable that is returned from `useMachine` will give you an object of the shape `{ currentState: [State](https://xstate.js.org/api/classes/state.html), send: [Interpreter.send](https://xstate.js.org/api/classes/interpreter.html#send)  }` - not sure how to type that properly because technically it returns a getter that will give you back this state.